components/prism-elisp.js

```diff
@@ -0,0 +1,192 @@
+// Functions to construct regular expressions
```

Comment:
Could you please wrap the whole file in `(function (Prism) {` and `}(Prism));` please ? That way, we won't pollute the global scope with vars and stuff.